.github/CODEOWNERS

```diff
@@ -1588,28 +1588,28 @@ src/platform/plugins/shared/discover/public/context_awareness/profile_providers/
 /x-pack/test/functional_with_es_ssl/apps/discover_ml_uptime/config.ts @elastic/ml-ui
 /x-pack/test/functional_with_es_ssl/apps/discover_ml_uptime/ml/ @elastic/ml-ui
 /x-pack/test/alerting_api_integration/spaces_only/tests/alerting/ml_rule_types/ @elastic/ml-ui
-/x-pack/test/alerting_api_integration/spaces_only/tests/alerting/transform_rule_types/ @elastic/ml-ui
+/x-pack/test/alerting_api_integration/spaces_only/tests/alerting/transform_rule_types/ @elastic/ml-ui @elastic/kibana-management
```

Comment:
Have you got all the `transform` files updated in the `CODEOWNERS` file here? For example, I don't see an edit for `x-pack/platform/plugins/private/transform @elastic/ml-ui`

Reply:
Oh good catch, I added it but the bot removed it again, because that part was auto-generated and needs to be set in the `kibana.jsonc` file of the plugin. Updated in [13e9a0c](https://github.com/elastic/kibana/pull/218808/commits/13e9a0c06903b647f43650cadfd2e60546bc6c2a).